presentation/testing/src/presentation-testing/IModelUtilities.ts

```diff
@@ -6,84 +6,89 @@
  * @module IModel
  */
 
-import { IModelDb, IModelJsFs, InformationPartitionElement, SnapshotDb } from "@itwin/core-backend";
+import { IModelDb, IModelJsFs, SnapshotDb } from "@itwin/core-backend";
 import { Id64String } from "@itwin/core-bentley";
-import { Code, CodeScopeProps, CodeSpec, ElementAspectProps, ElementProps, IModel, LocalFileName, ModelProps } from "@itwin/core-common";
+import { BisCodeSpec, Code, CodeScopeProps, CodeSpec, ElementAspectProps, ElementProps, LocalFileName, ModelProps } from "@itwin/core-common";
 import { IModelConnection, SnapshotConnection } from "@itwin/core-frontend";
 import path from "path";
-import { TEST_OUTPUT_DIR } from "./Helpers";
+import { getTestOutputDir } from "./Helpers";
 
+/**
+ * Interface for IModel builder pattern. Used for building IModels to test rulesets.
+ *
+ * @public
+ */
 export interface TestIModelBuilder {
   insertModel<TProps extends ModelProps>(props: TProps): Id64String;
   insertElement<TProps extends ElementProps>(props: TProps): Id64String;
   insertAspect<TProps extends ElementAspectProps>(props: TProps): void;
-  createCode(scopeModelId: CodeScopeProps, codeSpecName: string, codeValue: string): Code;
-  getUniqueModelCode(newModelCodeBase: string): Code;
+  /** codeValue has to be unique */
+  createCode(scopeModelId: CodeScopeProps, codeSpecName: BisCodeSpec, codeValue: string): Code;
```

Comment:
Missing docs on some functions.

Also, regarding the `codeValue` comment... It only needs to be unique within its scope, but I don't even think we need to document that. Maybe just point to documentation for codes?

Reply:
Added documentation on interface functions. Didn't add a reference to documentation, I thought it was unnecessary if that's the case. Do you think otherwise?

Reply:
Suggested a change in a review comment